src/atoms/Button/buttons.module.scss

```diff
@@ -191,12 +196,29 @@ $button-solid-background-tertiary-color:   #f08f5c;
 }
 
 .outline {
-  @include hover($button-outline-hover-background-color,
-  $button-outline-border-color,
-  $button-outline-hover-color);
-  background-color: $button-outline-background-color;
-  border-color: $button-outline-border-color;
-  color: $button-outline-color;
+  background-color: $brand-white;
+  color: $brand-black;
+  border: solid 2px $brand-black;
+  border-radius: 0;
+}
+
+.outline-orange {
+  background-color: $brand-white;
+  color: $brand-orange;
+  border: solid 2px $brand-orange;
+  border-radius: 0;
+}
+
+.filled {
```

Comment:
can you change this to `solid` to better reflect the language used in the zeplin?